sdk/js-query/src/query/utils.ts

```diff
@@ -0,0 +1,31 @@
+import * as elliptic from "elliptic";
+
+export function isHex(s: string) {
+  // TODO: even length?
+  return /^(0x)?[0-9a-fA-F]+$/.test(s);
+}
+
+export function hexToUint8Array(s: string): Uint8Array {
+  if (!isHex(s)) {
+    throw new Error(`${s} is not hex`);
+  }
+  return new Uint8Array(
```

Comment:
You may need to prepend a '0' if the length is odd?  That threw me off in the algorand sdk.